src/pages/components/image-with-caption.mdx

```diff
@@ -29,6 +30,16 @@ The Image with caption supports images at 1x1, 2x1, 4x3, and 16x9 aspect ratios.
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline (caption) | Short copy | Yes | 1 | XXS (40 / 55)|
+| Description | Long copy | No | 1 | L (150 / 200)|
+| Image | Media | Yes | 1 | - |
```

Comment:
Do we need an alt for the content model?

Reply:
That's a good question. I was wondering if we want to get into that here or if that's assumed with all image and video assets. i can certainly add it if you think it's helpful?

Reply:
I think if we're talking content model then I feel like it's important to call out. There is also supposed to be a long-form alternative text. Maybe just one alt is fine though. You could put short/long copy possibly. The way I tried to handle some of this was linking to the image component. I don't know if you want to handle some it in a similar way. Like it inherits the Image content model, and they can click to the image component. That way we don't have to adjust a hundred different content models if image decides to add something new.

Reply:
Ah yes! I forgot that you had a component page for 'Image'. I'll just link to that and then I don't think we need ALT tag here, but I think we need to add it to the 'Image' content model.